Packs/XSOARLabUpdates/Scripts/BuildSlackBlocksFromIndex/BuildSlackBlocksFromIndex.yml

```diff
@@ -5,13 +5,13 @@ args:
 - description: The last time the message was sent. Typically one week ago.
   name: last_run_str
   required: true
-comment: |-
-  Extracts the index.zip and filters new packs since the last run.
-  Builds the slack message for new packs.
+comment: 'Extracts the index.zip and filters new packs since the last run.
+
+  Builds the slack message for new packs.'
 commonfields:
   id: BuildSlackBlocksFromIndex
   version: -1
-dockerimage: demisto/unzip:1.0.0.19258
+dockerimage: demisto/unzip:1.0.0.100283
```

Comment:
@talzich can you look this over?

Reply:
@JudahSchwartz It looks like the script only uses Python's built-in ZipFile library and not the one from CommonScripts. Also, the dockerfile of unzip, and the one it starts from, does not contain the problamatic `pyminizip` library.

Reply:
@JudahSchwartz @talzich Are we good to merge?

Reply:
I don't see any issues